src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index = new Random().nextInt(Colors.values().length);
```

Comment:
your index has the same value on each getRandomColor() call. Let's better calculate index value within the method and make your Random instance a class field (initialized directly in the field or using a constructor)